Tests/FuzzilliTests/VariableMapTest.swift

```diff
@@ -59,30 +60,33 @@ class VariableMapTests: XCTestCase {
         XCTAssertEqual(m1, m2)
         
         // Add another 128 elements and compare with a new map built up in the opposite order
-        for i in 128..<256 {
+        for i in 127..<255 {
```

Comment:
Same, should be `128..<256`

Reply:
Changes reverted to the original implementation